terminal.cpp

```diff
@@ -67,9 +67,11 @@ Terminal::Terminal(QObject *parent)
     , iShowCursor(true)
     , iUseAltScreenBuffer(false)
     , iAppCursorKeys(false)
+    , iReplaceMode(false)
```

Comment:
nit: would prefer using 'm_' prefix, I've been doing that for all new (or changed) code. I should probably go on a mass style cleanup.

Reply:
Also prefer 'm_' more, but maybe better to do renamings separately. While at it, could reformat with astyle to get rid of inconsistencies.